index.bs

```diff
@@ -1482,9 +1514,13 @@ A {{WebTransportSendStream}} has the following internal slots.
    <td><dfn>`[[Transport]]`</dfn>
    <td class="non-normative">A {{WebTransport}} which owns this {{WebTransportSendStream}}.
   </tr>
+  <tr>
+   <td><dfn>`[[Sender]]`</dfn>
+   <td class="non-normative">An optional {{WebTransportSender}}, or null.
+  </tr>
   <tr>
    <td><dfn>`[[SendOrder]]`</dfn>
-   <td class="non-normative">An optional send order number, or null.
+   <td class="non-normative">An optional send order number, defaulting to 0.
```

Comment:
does this really need to be bundled?

Reply:
What do you mean? If you mean why not a separate PR, then I think the main desired outcome of this design was to simplify the sendOrder model, i.e. the old way becomes redundant.